prost-build/src/lib.rs

```diff
@@ -1124,12 +1124,16 @@ impl Config {
 
             debug!("Running: {:?}", cmd);
 
-            let output = cmd.output().map_err(|error| {
-                Error::new(
-                    error.kind(),
-                    format!("failed to invoke protoc (hint: https://docs.rs/prost-build/#sourcing-protoc): (path: {:?}): {}", &protoc, error),
-                )
-            })?;
+            let output = match cmd.output() {
+                Err(err) if ErrorKind::NotFound == err.kind() => {
+                    panic!("{}", protoc_not_found());
```

Comment:
should we not just forward this as an error as well? What is the value of panic here?

Reply:
The current behavior is to panic when `protoc` is not found by which. I agree forwarding the error is better. But only is an `unwrap()` of the error displays the explanation text.

Reply:
As @caspermeijn says, I was maintaining the existing functionality where a panic occurs because changing that felt out of scope of this change.
I've gone ahead and changed it to return an Err now.